scripts/config.js

```diff
@@ -10,8 +10,8 @@ const weexVersion = process.env.WEEX_VERSION || require('../packages/weex-vue-fr
 
 const banner =
   '/*!\n' +
-  ' * Vue.js v' + version + '\n' +
-  ' * (c) 2014-' + new Date().getFullYear() + ' Evan You\n' +
+  ` * Vue.js v${version}\n` +
+  ` * (c) 2014-${new Date().getFullYear()} Evan You\n` +
   ' * Released under the MIT License.\n' +
   ' */'
```

Comment:
Now that template literals are used, why mix them with string concatenations? It is better to replace the whole expression with a single template literal, IMHO.

Reply:
Multi-line template string will introduce unnecessary whitespaces, unless we adopt a weird code style that has no leading space in template string literals...

Reply:
Hmmm… My bad. Thought that was acceptable coding style (compared to mix usage of `'` & `` ` ``). Didn't check with existing codebase…